Header header
geometry_msgs/Twist cmd_vel
int32 mode
float32 a
float32 v
float32 vg
float32 p

